README.md

```diff
@@ -19,4 +20,5 @@ The main goal of Langium is to lower the barrier of creating a DSL / low-code pl
 
 ## How?
 
-Langium is available as [npm package](https://www.npmjs.com/package/langium). Please read the package documentation to get started.
+Langium is available as [npm package](https://www.npmjs.com/package/langium).
+Please read the [Local Development Guide](./CONTRIBUTING.md#local-development-guide) to get started.
```

Comment:
I wouldn't direct all users to that guide, only those who want to go deeper with the internals of Langium.
```suggestion

Please read the [Contribution Guide](./CONTRIBUTING.md) if you want to contribute to Langium.
```